### DISCUSSION TITLE
discord full screen share not working

### DISCUSSION THREAD

**Discussion by naiyanrahman** [prasanthrangan/hyprdots#927] [Q&A]
I tried to share my screen share in discord but can't share 
![240221_16h46m03s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/77504180/669f8fcb-f47b-468a-82e3-c74a7a467133)

**Comment by RookiexCookie** [comment]
yea Wayland and discord doesnt work that well together 
try using discord web

**Reply by ghost** [reply]
Yeah that would work or i use Vesktop as they have pretty much fixed all the Wayland bugs and have nice plugins.

**Comment by Marcus-Arcadius** [comment]
I've been using discord-screenshare which works perfectly and also has audio coming through the screen share as well
